src/login/login.cpp

```diff
@@ -718,8 +719,13 @@ void log_init(int argc, char** argv)
         {
             logFile = argv[i + 1];
         }
+
+	if (strcmp(argv[i], "--append-date") == 0)
```

Comment:
This whole if statement needs aligning aswell.

Reply:
Also probably makes more sense for this to be an `else if` because only one will match through each iteration of this loop anyway. No need to check if the first `if` matches.